src/main/java/org/ow2/proactive/procci/model/occi/metamodel/MixinBuilder.java

```diff
@@ -84,7 +84,7 @@ public MixinBuilder(MixinService mixinService, InstanceService instanceService,
         this.applies = new ArrayList<>();
         for (String apply : Optional.ofNullable(mixinRendering.getApplies()).orElse(new ArrayList<>())) {
             this.applies.add(
-                    InfrastructureKinds.getKind(apply).orElseThrow(() -> new SyntaxException(apply,"Kind")));
+                    InfrastructureKinds.getKind(apply).orElseThrow(() -> new SyntaxException(apply, "Kind")));
```

Comment:
if an apply is not found in the InfrastructureKinds map you throw a SyntaxException. Is that correct?